WebApplication1/Data/MovieDbContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.EntityFrameworkCore;
+using WebApplication1.Models;
+
+namespace WebApplication1.Data
+{
+    public class MovieDbContext : DbContext
+    {
+        public MovieDbContext(DbContextOptions<MovieDbContext> options) : base(options)
+        {
+
+        }
+        public DbSet<Movie> Movies { get; set; }
+        public DbSet<Theatre> Theatres { get; set; }
+        public DbSet<Worker> Workers { get; set; }
```

Comment:
For the DB configurations for relationships, you can either do them all in this file below the DB set in a builder method or have the configurations in different files and call them here using the builder. Please check the other comment for reference and more information